### DISCUSSION TITLE
Welcome to sylph Discussions!

### DISCUSSION THREAD

**Discussion by bluenote-1577** [bluenote-1577/sylph#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  For the maintainers, here are some tips 💡 for getting started with Discussions. We'll leave these in Markdown comments for now, but feel free to take out the comments for all maintainers to see.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by jolespin** [comment]
Migrating discussion about quantifying % of unclassified reads from https://github.com/bluenote-1577/sylph/issues/6#issuecomment-2386785933

Have you seen any research in determining a good cutoff for novelty in a dataset that warrants genome binning? For example, if only 25% of the reads are captured one will likely yield a few MAGs compared to if 80% of the reads are captured you might not getting any novel MAGs.

**Comment by bluenote-1577** [comment]
@jolespin Hmm that's a tricky question. I don't have an answer, since it's hard to determine what that unknown stuff is. It can be host DNA, viral DNA, etc...

**Comment by jolespin** [comment]
Not really a discussion topic but just saw Sylph got published in Nature Biotechnology.  Congrats!
https://www.nature.com/articles/s41587-024-02412-y

**Reply by bluenote-1577** [reply]
Thanks @jolespin !
